var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -294,6 +294,13 @@ def edit(self, spec, prefix):
             if "+single_node_gpu" in spec:
                 opts.extend(["--with-single-node-cuda"])
 
+        if "+rocm" in spec:
+            self._copy_arch_file("hip")
+            opts.append("--with-hip")
+            opts.append("--with-single-node-hip")
+            opts.append("--rocm-prefix")
+            opts.append(os.environ["ROCM_PATH"])
```

Comment:
The previous two lines should be combined as `opts.extend(["--rocm-prefix", os.environ["ROCM_PATH"]])` to ensure that the opts lines can be reordered without breaking them.

Reply:
Done